recipes/pot/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "POT" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "1c226e359e81ca08e593012b9022d64041318d0bd7aafcb30185f1c4f02304af" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy x.x
+    - scipy
+    - matplotlib
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - ot
+  requirements:
```

Comment:
I agree it won't break without it, but I thought it was best practice to put things there anyway if they're explicitly imported by `run_test.py`, since if they later get removed as runtime requirements then the test would break. At least that's what I thought [this comment](https://github.com/conda-forge/staged-recipes/blob/20d1705ea984a2cc1e190f13780b74266b3caa55/recipes/example/meta.yaml#L39) meant.